understanding/22/focus-appearance-minimum.html

```diff
@@ -133,6 +133,11 @@ <h4>Unusual shapes and gradients</h4>
                     <figcaption>The focused button with the non-contrasting areas removed, showing that it is a thick indicator that meets the requirements.</figcaption>
                 </figure>
 
+                <p>If an inline-link is broken over two lines the indicator can be split between them. In this case the link element is still treated as one instance.</p>
```

Comment:
A couple of minor suggestions, if that's OK:

- inline link doesn't appear to be hyphenated anywhere else in this repository, so remove hyphenation
- explicitly allow for links that span more than two lines
- use 'control' rather than 'instance' as 'instance' is not used anywhere else, and 'control' is defined.

Are there any other inline elements that might take keyboard focus? Should 'inline link' be 'inline element'?

```suggestion
                <p>If an inline link is broken over two or more lines the indicator can be split between them. In this case the link element is still treated as a single control.</p>
```